src/frontend/context/ManuellJournalførContext.tsx

```diff
@@ -484,13 +484,19 @@ const [ManuellJournalførProvider, useManuellJournalfør] = createUseContext(()
     };
 
     const kanKnytteJournalpostTilBehandling = () => {
-        return dataForManuellJournalføring.status !== RessursStatus.SUKSESS
-            ? false
-            : dataForManuellJournalføring.data.oppgave.oppgavetype === OppgavetypeFilter.BEH_SED &&
-                tilordnetInnloggetSaksbehandler() &&
-                toggles[ToggleNavn.brukEøs]
-              ? true
-              : !erLesevisning();
+        if (dataForManuellJournalføring.status !== RessursStatus.SUKSESS) {
+            return false;
+        }
+
+        if (
+            dataForManuellJournalføring.data.oppgave.oppgavetype === OppgavetypeFilter.BEH_SED &&
+            tilordnetInnloggetSaksbehandler() &&
+            toggles[ToggleNavn.brukEøs]
+        ) {
+            return true;
+        }
+
+        return !erLesevisning();
```

Comment:
Her lurer jeg egentlig litt på hvorfor vi ikke bare kan skrive (toggle for Eøs kan fjernes)
```
dataForManuellJournalføring.status === RessursStatus.SUKSESS && !erLesevisning() && tilordnetInnloggetSaksbehandler()
```

Tror egentlig ikke vi trenger både lesevisning og tilordnet-sjekken, fordi jeg tror siden havner i lesevisning om den ikke er tilordnet innlogget saksbehandler. Men mulig det er noe som jeg ikke skjønner, må være en grunn til at det er implementert sånn her 🤔

Reply:
Litt usikker på om jeg skjønner 100% hva som menes her, men tok en kjapp titt og ser ut som at ```tilordnetInnloggetSaksbehandler()``` allerede sjekker ```dataForManuellJournalføring.status === RessursStatus.SUKSESS``` og ```erLesevisning()``` sjekker både de to andre. Så enig at noe føles overflødig her 😊

Reply:
Støtter `dataForManuellJournalføring.status === RessursStatus.SUKSESS && !erLesevisning() && tilordnetInnloggetSaksbehandler()`, men vet ikke helt sikkert

Reply:
Lar den være som den er nå, og retter opp i pr hvor jeg fjerner toggle